### ISSUE TITLE
APLIC+IMSIC test

### ISSUE THREAD

**Issue by Ruofeng-Wang** [zero-day-labs/riscv-aia#3]
Hi, I am trying to play around with the APLIC and IMSIC designs, however, I noticed that there is no assertions in APLIC+IMSIC test. Does that mean the test will not check a failing or succeed condition?

**Comment by D3boker1** [comment]
Hi @Ruofeng-Wang!
The tests are undergoing many changes at the moment. We released the minimal version of the APLIC this week, and for now, we have been analyzing the waves with gtkwave (in the Python file, we have comments on the expected results). As some of the tests are repeated or incomplete, rewriting the tests is a task to which we must dedicate ourselves soon. When it is done, we will have assertions. 

The IPs were, however, implemented on a CVA6-based SoC ( [aplic](https://github.com/zero-day-labs/cva6/tree/feat/aia-aplic) and [full aia](https://github.com/zero-day-labs/cva6/tree/feat/aia-full) ), and we successfully ran Linux vanilla and Bao hypervisor + Linux vanilla.

Sorry for the inconvenience!
